identity/backend/src/main/resources/schema.sql

```diff
@@ -50,3 +52,26 @@ CREATE TABLE IF NOT EXISTS group_members (
 CREATE TABLE IF NOT EXISTS profiles(
     id bigint primary key, email varchar(100) unique,
     constraint fk_profile_users foreign key(id) references users(id) ON DELETE CASCADE);
+
+-- authority == role
+CREATE TABLE IF NOT EXISTS roles(
+    authority varchar(50) not null primary key,
+    description varchar(500)
+);
+
+CREATE TABLE IF NOT EXISTS permissions(
+    id bigint generated by default as identity(start with 1) primary key,
+    definition varchar(50) not null unique,
+    description varchar(50) not null
+);
+
+CREATE TABLE IF NOT EXISTS role_permissions(
+    role_authority varchar(50) not null,
+    permission varchar(50) not null,
+    constraint unique_rp unique(role_authority, permission)
+);
+
+insert into roles(authority, description)
+values ('ADMIN', 'administrator role');
+insert into roles(authority, description)
+values ('DEFAULT_USER', 'default user role');
```

Comment:
Let's remove the default insert statement or can't a user log in without having any roles assigned right now?